src/main/java/com/deloitte/elrr/CacheConfig.java

```diff
@@ -36,6 +38,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 						saml2.relyingPartyRegistrationRepository(relyingPartyRegistrations());
 					} catch (Exception e) {
 						// TODO Auto-generated catch block
+						log.error("Exception occurred: ", e);
```

Comment:
Not sure if we want to log the exception outright. Maybe just a description such as "SAML login failure occured" will ask James his opinion